src/compiler/crystal/command/playground.cr

```diff
@@ -34,6 +34,21 @@ class Crystal::Command
       end
     end
 
+    playground_url = "http://#{server.host || "localhost"}:#{server.port || "8080"}"
+
+    open_browser(playground_url)
```

Comment:
Please, put it behind a flag as mentioned in the https://github.com/crystal-lang/crystal/issues/7493#issuecomment-468737132.

Reply:
Can we get an official stance on whether this should be contained behind a flag or not?

With the latest changes suggested by @r00ster91 to use `Process.run` instead of `system`, that should hide any negative effects (e.g. error output) if the command fails.

I imagine the only thing left to discuss would be whether people would find it annoying that the browser window automatically opens to the Playground when using `crystal play`.

As a newcomer to the language, I certainly would prefer it to open automatically but I'm not sure if that is inconvenient to others.

Reply:
I think this definitely needs to be configurable. As already mentioned, this is a very opinionated features.
You should be able to run the playground for example as a background service and not try to open any http URL.

And I'd still question whether this flag should be active by default (is it opt-in or opt-out?). I know that opt-in would largely reduce it's usefulness, but again, I'm not even sure this needs to be implemented as a playground feature.

Reply:
I can't imagine situation of having such feature enabled without the ability to opt-out, c'mon... In any case I'd also question whether should it be turned on by default.

Reply:
I think the official should be to open the browser. Is better for newcomers and offers discoverability of the feature.

So I would go with an opt-out flag `-n --no-launch`.

Reply:
Added `-n` option in 1c90e117c1cb64b2cf7cec9ca4dc0d6d0f2e32e3. Works great.